stdlib/pkgutil.pyi

```diff
@@ -36,3 +36,6 @@ def walk_packages(
     path: Optional[Iterable[str]] = ..., prefix: str = ..., onerror: Optional[Callable[[str], None]] = ...
 ) -> Iterator[_ModuleInfoLike]: ...
 def get_data(package: str, resource: str) -> Optional[bytes]: ...
+
+if sys.version_info >= (3, 9):
+    def resolve_name(name: str) -> object: ...
```

Comment:
```suggestion
    def resolve_name(name: str) -> Any: ...
```

Generally, typeshed prefers false negatives over false positives. A return type of `object` means that you can't do anything with the return value, unless you add `assert isinstance(the_return_value, SomeType)`, which would cause false positives in any code using `resolve_names`.